pkg/retry/errors.go

```diff
@@ -37,6 +37,7 @@ var (
 	UnsupportedDMLMsgs = []string{
 		"Error 1062: Duplicate entry",
 		"Error 1406: Data too long for column",
+		"Error 1366: Incorrect datetime value",
```

Comment:
How about handing other `Error 1366` too? maybe you can search TiDB repo (`tidb/errno/errcode.go`) to got the meaning of this code.

Reply:
I guess it's no help keep retrying

https://github.com/pingcap/tidb/blob/0fd81a83e8a0f0c799e006b0be03205533ca2750/executor/insert_common.go#L287

seems in fact most errors could not resolved by retry 🤔

Reply:
> seems in fact most errors could not resolved by retry 

YES, but we don't know which errors could be resolved by retry, 😢